src/csharp/Microsoft.Spark/ML/Feature/Bucketizer.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Spark.ML.Feature
     /// will be thrown. The splits parameter is only used for single column usage, and splitsArray
     /// is for multiple columns.
     /// </summary>
-    public class Bucketizer : FeatureBase<Bucketizer>, IJvmObjectReferenceProvider
```

Comment:
Made FeatureBase implement IJvmObjectReferenceProvider instead.  Can move back to these classes if original design was better.

Reply:
this is OK.